.github/workflows/base-lambdas-reusable-deploy-all.yml

```diff
@@ -316,4 +316,32 @@ jobs:
       lambda_aws_name: UpdateUploadStateLambda
       lambda_layer_names: "core_lambda_layer"
     secrets:
-      AWS_ASSUME_ROLE: ${{ secrets.AWS_ASSUME_ROLE }}
\ No newline at end of file
+      AWS_ASSUME_ROLE: ${{ secrets.AWS_ASSUME_ROLE }}
+
+  deploy_data_collection_lambda:
+    name: Deploy data collection lambda
+    uses: ./.github/workflows/base-lambdas-reusable-deploy.yml
+    with:
+      environment: ${{ inputs.environment}}
+      python_version: ${{ inputs.python_version }}
+      build_branch: ${{ inputs.build_branch}}
+      sandbox: ${{ inputs.sandbox }}
+      lambda_handler_name: data_collection_handler
+      lambda_aws_name: DataCollectionLambda
+      lambda_layer_names: "core_lambda_layer,data_lambda_layer"
```

Comment:
Do both these lambdas require polars?

Reply:
Yes, I have used polars in the DataCollection lambda for a few metrics that are a bit tricky and need some calculation, for example, "average number of records per patient per each ODS code"